app/src/main/res/values/strings.xml

```diff
@@ -451,6 +453,17 @@
     <!-- Preference for using following device theme -->
     <string name="preference_follow_device_theme">Follow device theme</string>
 
+    <!-- Gestures Preferences-->
+    <string name="preference_gestures_group_name">Gestures</string>
```

Comment:
Is `preference_gestures_group_name` used anywhere? It seems preferences_gestures is defined for preference category name.

Reply:
Eagle eye! Thanks!
Wasn't sure where to put the name, how to name the string.
Seems like `preferences_gestures` won and is more in line with the titles of the other categories.